Glide fix

ljagiello opened the pull request:
Error:
```
[ERROR] Update failed for github.com/Sirupsen/logrus: The Remote does not match the VCS endpoint
[ERROR] Failed to install: The Remote does not match the VCS endpoint
```

github.com/Sirupsen/logrus is now github.com/sirupsen/logrus.

jaxxstorm commented:
This is a continuing problem with the logrus package. Can you fix it like so? https://github.com/sirupsen/logrus/issues/553#issuecomment-306591437

ljagiello commented:
Sure

Commit:
Glide fix - The Remote does not match

Error:
```
[ERROR] Update failed for github.com/Sirupsen/logrus: The Remote does not match the VCS endpoint
[ERROR] Failed to install: The Remote does not match the VCS endpoint
```

github.com/Sirupsen/logrus is now github.com/sirupsen/logrus.
